### PR TITLE
Yuliia/wishlist create update

### PR DESCRIPTION
**Pull request by ChebotarYuliia**

### CODE REVIEW

**Review by ValeriaRbk — APPROVED** [review]

**Review by npelenskii — APPROVED** [review]

**Review by ShevtsovVolodymyr — APPROVED** [review]

File: controllers/wishlist.controllers.js [hunk 1]
```diff
@@ -49,9 +50,7 @@ async function getAllWishlistItems(req, res, next) {
     return next(error);
   }
 
-  return res.status(200).json({
-    ...wishlistItems,
-  });
+  return res.status(200).json(...wishlistItem);
```
Comment (AntonDeXy) [inline controllers/wishlist.controllers.js]: move res.status(200)... into try block and declare wishlistItem variable in try block too, using const keyword

File: routes/wishlist.routes.js [hunk 2]
```diff
@@ -15,9 +15,9 @@ router.post(
   wishlistController.createWishlistItem,
 );
 router.get(
-  '/',
+  '/getitem/:id',
```
Comment (AntonDeXy) [inline routes/wishlist.routes.js]:
```suggestion
  '/get-item/:id',
```

it will improve readability :)

Comment (ChebotarYuliia) [reply routes/wishlist.routes.js]: done

File: controllers/wishlist.controllers.js (hunk 1 above)
Comment (ChebotarYuliia) [reply controllers/wishlist.controllers.js]: done

**Pull request merged**